### ISSUE TITLE
Whitespace between moustache braces will not render data

### ISSUE THREAD

**Issue by bovisp** [KingPixil/moon#23]
### Description

If there is any whitespace between the moustache braces, the information within the braces will not render. Some (like myself) like to incorporate a space between `{{}}` and the data to make the code more readable.

### Steps to Reproduce

Create a new Moon instance and within the data object, add a message:

```
var app1 = new Moon({
  el: "#app1",
  data: {
    message: "Hello Moon!"
  }
});
```

and within your HTML:

```
<p>{{ message }}</p>
```

### Expected behavior

`{{ message }}` should be replaced by "Hello Moon!"

### Actual behavior

All you see in your HTML is "{{ message }}".

**Comment by KingPixil** [comment]
I'll be making the mustache compiler more flexible in v0.9.0

**Comment by KingPixil** [comment]
Fixed in 838e7a2b
